src/app/layouts/expansion-panel/expansion-sample-1/expansion-sample-1.component.scss

```diff
@@ -1,14 +1,11 @@
-.igx-expansion-panel {
+:host {
+    display: block;
     font-family: Verdana;
     margin: 24px;
     width: 420px;
     box-shadow: 0 1px 5px 0 rgba(0, 0, 0, .26), 0 2px 2px 0 rgba(0, 0, 0, .12), 0 3px 1px -2px rgba(0, 0, 0, .08);
 }
 
-.igx-expansion-panel__header--expanded {
```

Comment:
These are mentioned in the topic - if you delete them from the samples, make sure to adjust the topic as well.

Reply:
Instead of simply adjusting these, I have removed the specific scss code snippet as well as the above comment related to the styles (the snippet can be seen in the stackblitz). Additionally I have relocated Both **Styling** and  **Styling the header** sections.

Reply:
`.igx-expansion-panel__header` and `.igx-expansion-panel__body` are no longer mentioned as it is better to use the component's tag selectors as in the samples.